WooCommerce/Classes/ViewRelated/AppCoordinator.swift

```diff
@@ -64,7 +66,7 @@ final class AppCoordinator {
         self.featureFlagService = featureFlagService
         self.purchasesManager = purchasesManager
         self.switchStoreUseCase = SwitchStoreUseCase(stores: stores, storageManager: storageManager)
-
+        self.inAppPurchasesPlanManager = inAppPurchasesPlanManager
```

Comment:
If you take the above suggestion, this can be removed and the `purchasesManager` made non-optional.